Makefile

```diff
@@ -0,0 +1,68 @@
+#!make
+ROOT_DIR:=$(CURDIR)
+include .env
+
+### Adding a chain
+.PHONY: add-chain
+add-chain:
+	go run ./add-chain
+	# go run ./add-chain check-rollup-config
+	mkdir -p ./superchain/extra/genesis/$(SUPERCHAIN_TARGET)
+	cd $(MONOREPO_DIR) && go run ./op-chain-ops/cmd/registry-data \
+		--l2-genesis=$(GENESIS_CONFIG) \
+		--bytecodes-dir=$(ROOT_DIR)/superchain/extra/bytecodes \
+		--output=$(ROOT_DIR)/superchain/extra/genesis/$(SUPERCHAIN_TARGET)/$(CHAIN_SHORT_NAME).json.gz
+	make codegen
+
+promote-to-standard:
+	go run ./add-chain promote-to-standard --chain-id=${chain-id}
```

Comment:
Should the arg be changed to `$(CHAIN_ID)` to read the env var?

Reply:
I don't think we have that in the `.env.example` currently (we read it from the rollup.json file during `add-chain`).